### PR TITLE
vuln(NSWG-ECO-508): fix vulnerable/fixed versions

### PR DESCRIPTION
**Pull request by knqyf263**
According to 
- https://www.npmjs.com/advisories/1432
- https://github.com/advisories/GHSA-884p-74jh-xrg2

### CODE REVIEW

File: vuln/npm/508.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   "cves": [
     "CVE-2019-15599"
   ],
-  "vulnerable_versions": "*",
-  "patched_versions": null,
+  "vulnerable_versions": "<1.2.2",
+  "patched_versions": ">=1.2.2,
```
Comment (nschonni) [inline vuln/npm/508.json:18]:
```suggestion
  "patched_versions": ">=1.2.2",
```

Comment (knqyf263) [reply vuln/npm/508.json]: Oops. I merged your suggestion. Thanks.

**Comment by knqyf263** [comment]
Any updates?

**Comment by DanielRuf** [comment]
We might have to change the text in the `recommendation` field.

**Comment by knqyf263** [comment]
@DanielRuf Thanks. I've updated it.

**Comment by lirantal** [comment]
Looks good.
Thanks @knqyf263 and @DanielRuf

**Pull request merged**